src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -6664,6 +6664,20 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
             arg.loc = loc;
             arg.m_value = s_var;
             fn_args.push_back(al, arg);
+        } else if (attr_name == "join") {
+            if (args.size() != 1) {
+                throw SemanticError("str.join() takes one argument",
+                    loc);
+            }
+            fn_call_name = "_lpython_str_join";
+            ASR::call_arg_t str_var;
+            str_var.loc = loc;
+            str_var.m_value = s_var;
+            ASR::call_arg_t passed_int;
+            passed_int.loc = loc;
+            passed_int.m_value = args[0].m_value;
```

Comment:
I just noticed this. Here, I think `passed_int` should be named `list_of_str` (or something similar). (@Agent-Hellboy  I remember we were experimenting with it by passing an integer.)

I think you can fix the above in your new PR https://github.com/lcompilers/lpython/pull/2364. You can just make the rename a separate commit.

Reply:
Go ahead and create an issue, so that we don't forget.

Reply:
created https://github.com/lcompilers/lpython/issues/2365